README.md

```diff
@@ -157,6 +153,42 @@ if time.time() > client.token_expires_at:
     expires_at = refresh_response["expires_at"]
 ```
 
+## Automatic token refresh
```

Comment:
I think it would be nice to have a tiny quick start and then we can link to our docs so we aren't trying to maintain examples in two places!

Reply:
Update: in #613 i just linked to the docs to avoid maintaining instructions in several places. so I'll rebase this pr after we review merge that other pr - Or I'll rebase that one against this if this gets merged first. but the goal is to not dump all docs into the readme rather keep them confined to the docs and link to them.